doc-surrealdb_versioned_docs/version-1.x/introduction/start.mdx

```diff
@@ -52,7 +52,7 @@ Here's what each segment of this command does:
 
 [SurrealQL](/docs/surrealdb/surrealql/) is the query language for SurrealDB which can be used to query data from your SurrealDB database. While this is not a requirement for getting started, it is helpful to familiarise yourself with some commands.
 
-There are a few different ways to use SurrealQL, including using the [Surrealist](https://Surrealist.app) or using the SurrealDB CLI.
+There are a few different ways to use SurrealQL. The [Surrealist app](https://Surrealist.app) is a GUI that can be used in a web browser or as a desktop app, and the [SurrealDB CLI](https://surrealdb.com/docs/surrealdb/cli/) can be used from the command-line.
```

Comment:
I think a single paragraph might look a bit too wordy so broken it down a bit more. Let me know your thoughts @matthewjnield 
```suggestion
There are two main ways to use SurrealQL:

1. **Surrealist**: A graphical user interface (GUI) that can be accessed via a web browser or used as a desktop application.
2. **SurrealDB CLI**: A command-line interface that allows you to interact with SurrealQL directly from the terminal.

For more information, you can visit [Surrealist](https://Surrealist.app), and the [SurrealDB CLI documentation](/docs/surrealdb/cli/).```